packages/app/src/components/InfoBox.tsx

```diff
@@ -9,9 +9,10 @@ import { NO_VALUE } from 'constants/placeholder'
 type InfoBoxRowProps = {
 	children?: React.ReactNode
 	title: string
-	value: React.ReactNode
 	keyNode?: React.ReactNode
-	valueNode?: React.ReactNode
+	textValue?: string
+	textValueSpan?: React.ReactNode
```

Comment:
I've updated the name for now but I think this component can be improved more, we'll make it part of the on going component refactor instead of holding up this PR too much.